src/main/java/com/jvm_bloggers/admin_panel/mailing/MailingAddressUniquenessValidator.java

```diff
@@ -0,0 +1,46 @@
+package com.jvm_bloggers.admin_panel.mailing;
+
+import com.jvm_bloggers.core.mailing.domain.MailingAddressRepository;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.markup.html.form.FormComponent;
+import org.apache.wicket.markup.html.form.validation.AbstractFormValidator;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.apache.wicket.util.lang.Args;
+
+import java.util.Optional;
+
+public class MailingAddressUniquenessValidator extends AbstractFormValidator {
```

Comment:
No, because id/edit fields are shared per all rows, form model is loaded after click on 'Edit', so validators are already there...